_includes/master/manifests/calico.yaml

```diff
@@ -31,17 +31,27 @@ calico.yaml acccepts the following include flags:
 {% include {{page.version}}/manifests/calico-etcd-secrets.yaml %}
 ---
 {%- elsif include.datastore == "kdd" and include.typha == "true" %}
+{% include {{page.version}}/manifests/rbac-kdd-calico.yaml %}
```

Comment:
I think the idea here is right, but we should reconsider the approach.

Note how the `calico-config.yaml` include is done - can we do something similar here?

e.g., a single include:

```
{% include {{page.version}}/manifests/rbac.yaml datastore=include.datastore network=include.network %}
```

I think this model will be conceptually simpler, and easier to maintain moving forward.

Reply:
OK sure thing...let me look into this. It was something I thought of but the sources seemed quite different without much in common. That doesn't mean we shouldn't put them in a single file, however.